assignment_day_11/src/components/Button.js

```diff
@@ -0,0 +1,33 @@
+import React, {Component} from 'react';
+import {View, Text, StyleSheet, TouchableOpacity} from 'react-native';
+import colors from '../constants/colors';
+
+class Button extends Component {
```

Comment:
we can convert this into a functional component. why you created this as class ?

Reply:
Just for practice, I created one component with the functional approach and one with the class approach.
Will make sure it is not repeated in the future